index.html

```diff
@@ -14,6 +19,153 @@
+          </div>
+
+        </div>
+
+        <!-- this is the start of the past schooling section -->
+        <div class="school-xp">
+          <h2 class='section-head'>Education</h2>
+
+          <!-- this is the start of the first past school card -->
+          <div class="card-wrap school-card">
+            <h3>Bowling Industry Mangement & Technology</h3>
+            <h3 class='secondary'>Vincennes University</h3>
+            <p class='date'>1994 - 2001</p>
+          </div>
+
+        </div>
+
+      </section>
+
+      <!-- this is the start of the right side of the main page -->
+      <section class="column">
+        <h2 class='section-head'>Skills</h2>
+
+        <!-- skills section cards -->
+        <div class="card-wrap skills-card">
+          <div class="skills-blocks">
+            <span class="skill-item">Data Analysis</span>
+            <span class="skill-item">Copywriting</span>
+            <span class="skill-item">Foreign Languages</span>
+            <span class="skill-item">Accounting</span>
+            <span class="skill-item">Computer Languages</span>
+            <span class="skill-item">Mathematics</span>
+            <span class="skill-item">Graphic Design</span>
+            <span class="skill-item">Bookkeeping</span>
+            <span class="skill-item">SEO / SEM Marketing</span>
+            <span class="skill-item">Event Planning</span>
+            <span class="skill-item">Communication</span>
+            <span class="skill-item">Tardyness</span>
+          </div>
+        </div>
+
+        <h2 class='section-head'>Organizations</h2>
+
+        <!-- organizations section cards -->
+        <div class="card-wrap org-card">
+          <h3>Tickle Monsters United</h3>
+          <p class='date'>1901 - Present</p>
+    
+          <h3>Y2K Is The End!</h3>
+          <p class='date'>1999 - 2000</p>
+    
+          <h3>Super Good Time Anonyomous</h3>
+          <p class='date'>2019 - Forever</p>
+        </div>
+
+        <h2 class='section-head'>Honours and Awards</h2>
+          
+        <!-- honors and awards card section -->
+        <div class="card-wrap honorary-card">
+          <h3>Four Touchdowns Single Game (1966)</h3>
+          <h3 class='secondary'>Polk High Panthers</h3>
+    
+          <h3>Diabetes Award (2004)</h3>
+          <h3 class='secondary'>The Dundies</h3>
+    
+          <h3>D.E.N.N.I.S. System Completion (2009)</h3>
+          <h3 class='secondary'>Dennis Reynolds</h3>
+        </div>
+
+        <h2 class='section-head'>Languages</h2>
+
+        <!-- how you speakith like that -->
+        <div class="card-wrap lang-card">
+          <p><span>English</span><span class="fluentBox english"></span><span>Fluent</span></p>
+          <p><span>Spanish</span><span class="fluentBox spanish"></span><span>Fluent</span></p>
+          <p><span>French</span><span class="fluentBox french"></span><span>Fluent</span></p>
+        </div>
+
+      </section>
+    </div>
+  </div>
```

Comment:
no navigation, but you have so much other stuff, working links, other sections, etc